topics/valkey-json.md

```diff
@@ -0,0 +1,261 @@
+
+* There are five existing Valkey and Redis OSS ACL categories that are updated to include the new JSON commands: @read, @write, @fast, @slow and @admin. The following table indicates the mapping of JSON commands to the appropriate categories.
+
+## JSON ACL Command Mapping
+
+| JSON Command       | @read | @write | @fast | @slow | @admin |
+|--------------------|:-----:|:------:|:-----:|:-----:|:------:|
+| JSON.ARRAPPEND     |       |   y    |   y   |       |        |
+| JSON.ARRINDEX      |   y   |        |   y   |       |        |
+| JSON.ARRINSERT     |       |   y    |   y   |       |        |
+| JSON.ARRLEN        |   y   |        |   y   |       |        |
+| JSON.ARRPOP        |       |   y    |   y   |       |        |
+| JSON.ARRTRIM       |       |   y    |   y   |       |        |
+| JSON.CLEAR         |       |   y    |   y   |       |        |
+| JSON.DEBUG         |   y   |        |       |   y   |   y    |
+| JSON.DEL           |       |   y    |   y   |       |        |
+| JSON.FORGET        |       |   y    |   y   |       |        |
+| JSON.GET           |   y   |        |   y   |       |        |
+| JSON.MGET          |   y   |        |   y   |       |        |
+| JSON.MSET          |       |   y    |       |   y   |        |
+| JSON.NUMINCRBY     |       |   y    |   y   |       |        |
+| JSON.NUMMULTBY     |       |   y    |   y   |       |        |
+| JSON.OBJKEYS       |   y   |        |   y   |       |        |
+| JSON.OBJLEN        |   y   |        |   y   |       |        |
+| JSON.RESP          |   y   |        |   y   |       |        |
+| JSON.SET           |       |   y    |       |   y   |        |
+| JSON.STRAPPEND     |       |   y    |   y   |       |        |
+| JSON.STRLEN        |   y   |        |   y   |       |        |
+| JSON.TOGGLE        |       |   y    |   y   |       |        |
+| JSON.TYPE          |   y   |        |   y   |       |        |
+
+## Command Syntax
+
+Most commands require a key name as the first argument. Some commands also have a path argument. The path argument defaults to the root if it's optional and not provided.
+
+**Notation**
+* Required arguments are enclosed in angle brackets. For example: <key> 
+* Optional arguments are enclosed in square brackets. For example: [path]
+* Additional optional arguments are indicated by an ellipsis ("…"). For example: [json ...]
+
+## Path Syntax
+Redis JSON supports two kinds of path syntaxes:
+
+* **Enhanced syntax** – Follows the JSONPath syntax described by [Goessner](https://goessner.net/articles/JsonPath/), as shown in the following table. We've reordered and modified the descriptions in the table for clarity.
+* **Restricted syntax** – Has limited query capabilities.
+
+If a query path starts with `$`, it uses the enhanced syntax. Otherwise, the restricted syntax is used.
```

Comment:
```suggestion
If a query path starts with `$`, it uses the enhanced syntax. Otherwise, the restricted syntax is used. It is recommended you use the enhanced syntax for new development.
```